project-base/tests/FrontendApiBundle/Functional/Order/GetOrderAsAuthenticatedCustomerUserTest.php

```diff
@@ -61,7 +67,10 @@ private function getOrderDataForCurrentlyLoggedCustomerUserProvider(): array
                 $order->getUuid(),
                 [
                     'status' => $order->getStatus()->getName(),
-                    'totalPriceWithVat' => $order->getTotalPriceWithVat()->getAmount(),
+                    'totalPriceWithVat' => $this->moneyFormatterHelper->getMoneyFormattedAsDecimalWithFractionDigitsByCurrencySetting(
```

Comment:
In tests we are using `FunctionalTestCase::getPriceWithVatConvertedToDomainDefaultCurrency()` for getting right price for output. If you need same function but add formatting I would expect to create new method in same class, something like`getFormattedPriceWithVatConvertedToDomainDefaultCurrency` which could be alias of some helper, but I think that it should be here.

Reply:
I do not see a good reason to (not) doing so.

It may comes handy when writing a new test and having this method right in `FunctionalTestCase`. On other hand I am a bit worried if it is the right place to be. In some time the `FunctionalTestCase` will became something like `ProductFacade`.